test/serializers/attributes_test.rb

```diff
@@ -23,12 +23,6 @@ def test_attributes_with_fields_option
                      @profile_serializer.attributes(fields: [:name]))
       end
 
-      def test_required_fields
-        assert_equal({name: 'Name 1', description: 'Description 1'},
-                     @profile_serializer.attributes(fields: [:name, :description], required_fields: [:name]))
-
-      end
-
```

Comment:
As explained, this is not a breaking change, as this feature was not
actually a frature but merely a hack that happened to be exposed.

Le mercredi 2 septembre 2015, Benjamin Fleischer <notifications@github.com>
a écrit :

> In test/serializers/attributes_test.rb
> <https://github.com/rails-api/active_model_serializers/pull/1103#discussion_r38569520>
> :
>
> > @@ -23,12 +23,6 @@ def test_attributes_with_fields_option
> >                       @profile_serializer.attributes(fields: [:name]))
> >        end
> >
> > -      def test_required_fields
> > -        assert_equal({name: 'Name 1', description: 'Description 1'},
> > -                     @profile_serializer.attributes(fields: [:name, :description], required_fields: [:name]))
> > -
> > -      end
> > -
>
> I'd just prefer to keep breaking changes out of refactoring pr...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails-api/active_model_serializers/pull/1103/files#r38569520>
> .
>


-- 
Lucas Hosseini
lucas.hosseini@gmail.com